README.md

```diff
@@ -59,6 +59,43 @@
 
 [![Install](https://i.imgur.com/1NJHIbH.gif)](https://github.com/qdm12/latexdevcontainer#setup-for-a-project)
 
+### Run inside GitHub Codespaces
+
+Remember, if you are running this devcontainer inside a GitHub Codespace, you have to uncomment the following
+line from `.devcontainer/docker-compose.yml`:
+
+```diff
+version: "3.2"
+
+services:
+  vscode:
+    build: .
+    image: latexdevcontainer
+    volumes:
+      - ../:/workspace
+      # Docker socket to access Docker server
+      - /var/run/docker.sock:/var/run/docker.sock
+      # SSH directory
+-     - ~/.ssh:/root/.ssh
++     # - ~/.ssh:/root/.ssh
+      # For Windows without WSL, a copy will be made
+      # from /tmp/.ssh to ~/.ssh to fix permissions
+-     - ~/.ssh:/tmp/.ssh:ro
++     # - ~/.ssh:/tmp/.ssh:ro
+      # Shell history persistence
+-     - ~/.zsh_history:/root/.zsh_history:z
++     # - ~/.zsh_history:/root/.zsh_history:z
+      # Git config
+-     - ~/.gitconfig:/root/.gitconfig
++     # - ~/.gitconfig:/root/.gitconfig
```

Comment:
Actually thinking more about this, should we flip this around?

Comment them in .devcontainer/docker-compose.yml and not have this section in the readme?
That would make more sense I think.

I'll see what I can do with https://github.com/qdm12/devtainr as well so it can uncomment those fields when setting up a repo 😉  Created https://github.com/qdm12/devtainr/issues/68